ross/units.py

```diff
@@ -63,10 +63,13 @@ def inner(*args, **kwargs):
                 # For now, we only return the magnitude for the converted Quantity
                 # If pint is fully adopted by ross in the future, and we have all Quantities
                 # using it, we could remove this, which would allows us to use pint in its full capability
-                try:
-                    base_unit_args.append(arg_value.to(units[arg_name]).m)
-                except AttributeError:
-                    base_unit_args.append(Q_(arg_value, units[arg_name]).m)
+                if arg_value is None:
+                    pass
```

Comment:
You still need to add the `arg_value` to the `base_unit_args` list to keep the consistency (args that goes into `check_units` needs to come back).

The code would not pass the following test:
```python
def test_positional_arguments():
    mat = Material('material', 7000, None, 81.2e9, 0.27)
    assert mat.rho == 7000
    assert mat.E == 206248000000.0
    assert mat.G_s == 81.2e9
    assert mat.Poisson == 0.27
```

I recommend adding a check for `arg_value` on line 62:
```python
        for arg_name, arg_value in zip(args_names, args):
            if arg_name in units and arg_value is not None:
                # For now, we only return the magnitude for the converted Quantity
                # If pint is fully adopted by ross in the future, and we have all Quantities
                # using it, we could remove this, which would allows us to use pint in its full capability
                try:
                    base_unit_args.append(arg_value.to(units[arg_name]).m)
                except AttributeError:
                    base_unit_args.append(Q_(arg_value, units[arg_name]).m)
            else:
                base_unit_args.append(arg_value)
```

Reply:
yes, I just noticed the same problem here. Gonna fix it again